benchmark/opperf/rules/default_params.py

```diff
@@ -152,7 +158,13 @@
                    "dim1": DEFAULT_DIM_1,
                    "dim2": DEFAULT_DIM_2,
                    "block_size": DEFAULT_BLOCK_SIZE,
-                   "args": DEFAULT_ARGS}
+                   "args": DEFAULT_ARGS,
+                   "grid": DEFAULT_GRID,
+                   "data_bilinearsampler": DEFAULT_DATA_BILINEAR,
```

Comment:
Seems there are very operator specific. What if these arguments change in mxnet 2.0 does it break this opperf utility. Is there a way to generalize these op specific params, like passing them in as general `args` param?

Reply:
If these arguments change (in terms of data type or arg name) the opperf utility will fail, as is the case for all of the more general args used in this file. I could use a general `args` param to pass in the op-specific arguments and establish a mapping between keywords and values on the fly, which might be more resilient to potential changes in arg structure (such as arg order or name changes). @ChaiBapchya any thoughts?

Reply:
As discussed offline @apeforest this should be covered as part of https://github.com/apache/incubator-mxnet/issues/17515 where changes made to ops shouldn't break OpPerf.